modules/imaging_uploader/php/imaging_uploader.class.inc

```diff
@@ -369,6 +369,25 @@ class Imaging_Uploader extends \NDB_Menu_Filter_Form
                 " .tgz, .tar.gz or .zip."
             );
         }
+        ///////////////////////////////////////////////////////////////////////
+        //////// Make sure there's no existing identical entry ////////////////
+        //////// for "new" scans //////////////////////////////////////////////
+        ///////////////////////////////////////////////////////////////////////
+        $updateFile = $args['values']['overwrite'] ? true : false;
+        $id         = $db->pselectOne(
+            "SELECT 
+                IF (InsertionComplete=0 AND 
+                (ISNULL(Inserting) OR Inserting=0), UploadID, NULL) as UploadID 
+                FROM mri_upload WHERE UploadLocation LIKE :ul",
+            array('ul' => "%$file_name")
+        );
+
+        if (!$updateFile && !empty($id)) {
+            array_push(
+                $errors['mri_file'],
+                "This file has already been uploaded! Please refresh your browser."
```

Comment:
Should "refresh your browser" be part of the error message?

Reply:
i added it because this error is typically caught on the front end. i added the browser refresh suggestion in order to update the react component's state. not opposed to removing it though

Reply:
If you look around here there's some sort of auto-refresh redirect thing. I'm not sure if it's exactly what you want but if a refresh is necessary I think I like this better than asking the user to do it.

https://github.com/aces/Loris/blob/33de6999da1748f20ba23c17d77570052bd5a6e4/modules/issue_tracker/jsx/IssueForm.js#L330